src/content/post/20160405_bbh.md

```diff
@@ -0,0 +1,51 @@
++++
+date        = "2016-04-06T08:51:09+02:00"
+title       = "Brown bag happy-hour, April 5th 2016"
+tags        = ['drush8', 'hugo', 'user experience', 'design']
+topics      = ['Brown bag happy hour']
+description = "First spring BBH"
+slug        = 'brown-bag-happy-hour'
+author      = "Marcello Gorla"
++++
+[SparkFabrik]: http://www.sparkfabrik.com/  "SparkFabrik"
+[Fast Co Design]: http://www.fastcodesign.com/3058517/frog-will-have-no-more-chief-creative-officers "Fast Co Design"
+
+In the first BBH of 2016 spring Marcello Testi discussed about Drush Make related issues within v8 of Drush, and Paolo Mainardi introduced the Hugo static site generator to the team in order to kickstart the [SparkFabrik] Tech Blog. We also talked a little about design thanks to our weekly news collection.
+
+<!--more-->
+
+# Weekly News
+
+This week we discussed mainly design-related news, brought by Marcello Gorla.
+
+### Adobe XD
+
+![Adobe XD](/posts/2016_adobexd.jpg)
+
+Originally know as “Project Comet", Adobe has released the preview of [Adobe XD](http://blogs.adobe.com/creativecloud/introducing-adobe-experience-design-cc-preview/), it's own tool oriented to design and prototype user experience projects. The tool it's an early stage release that still does not offer a full stack of features as we are used to with more advanced (and mature) softwares like Sketch or Adobe Photoshop, but there's a [lot of stuff moving on](https://www.youtube.com/playlist?list=PLD8AMy73ZVxXvBQcZAnOcu57JCFxLi7bQ) and we think it's worth trying.
```

Comment:
> oriented to design and prototype user experience projects

one of those two variants:

`intended for design and prototyping of user experience projects` or
`dedicated to design and protoyping of ...`